src/app.js

```diff
@@ -1,21 +1,50 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const fs = require('fs');
+
+const [currentFile, path] = process.argv.slice(2);
+
+fs.stat(path, (statErr, stats) => {
+  if (statErr && statErr.code === 'ENOENT') {
+    fs.rename(currentFile, path, (renameErr) => {
+      if (renameErr) {
+        // eslint-disable-next-line no-console
+        console.log(`Such directory '${path}' does not exist.`);
```

Comment:
I think it's better to throw an error
![image](https://user-images.githubusercontent.com/96409563/209577966-87817d4e-cace-46be-a9a9-d5084b7679f1.png)

Reply:
I changed everything!
Thanks for your help